src/appliance-onboarding-script/run.sh

```diff
@@ -129,7 +129,7 @@ python3.8 -m venv .temp/.env || fail
 activate_python_venv
 python -m pip install --upgrade pip || fail
 python -m pip install -r ./appliance_setup/dependencies || fail
-python ./appliance_setup/run.py "$1" "$2" "${3:-INFO}"
+python ./appliance_setup/run.py "$1" "$2" "${3:-INFO}" "${4:-None}"
```

Comment:
can we test this script for regression before this change. It will break due to the az cli bug. but we should ensure we do not introduce some new issue and that once the az cli issue is resolved, things should resume to work

Reply:
can we also add a comment block / TODO in the code file here denoting that the work for automation testing is yet to be done for the bash script?

Reply:
Done